conf/routes

```diff
@@ -388,6 +388,7 @@ GET            /api/files/:id/blob
 
 # deprecrated use DELETE
 POST           /api/files/:id/remove                                                    @api.Files.removeFile(id: UUID)
+POST           /api/files/bulkRemove                                                    @api.Files.bulkDeleteFiles()
```

Comment:
since this is a new API endpoint can you update the swagger documentation as well.

Reply:
I added a new entry in the swagger docs